Code/Framework/AzFramework/AzFramework/Windowing/NativeWindow.h

```diff
@@ -175,7 +175,7 @@ namespace AzFramework
 
             virtual void SetWindowTitle(const AZStd::string& title);
             virtual WindowSize GetClientAreaSize() const;
-            virtual void ResizeClientArea(WindowSize clientAreaSize);
+            virtual void ResizeClientArea(WindowSize clientAreaSize, bool ignoreScreenSizeLimit);
```

Comment:
Instead of updating this function to accept a bool, how about adding a structure for accepting window position parameters, so that the API of this function wouldn't have to change, if for some reason a another window position flag is needed.
```suggestion
            struct WindowPosOptions
            {
                bool m_ignoreScreenSizeLimit{};
            };
            virtual void ResizeClientArea(WindowSize clientAreaSize, const WindowPosOptions&);
```

That way this can still be called in an EBus as
```c++
AzFramework::WindowRequestBus::Event(windowHandle, &AzFramework::WindowRequestBus::Events::ResizeClientArea, clientAreaSize, WindowPosOptions{ ignoreScreenSizeLimit });
```

If for some reason, we need to add other options for Windows, Mac or Linux, only the handlers, that need to use the new options would need to be updated